### ISSUE TITLE
The VMWware Tools power-on script did not run successfully 

### ISSUE THREAD

**Issue by jtorri** [GNS3/gns3-gui#2523]
Windows 10 Prof 64bit. VMWare Workstation 14.1.2 build-8497320
Did full clone of GNS3 2.1.6 and upgraded it to 2.1.8 via provided upgrade menu option.
Also modified .gns3 project files and gns3 application ini files from 2.1.6 to 2.1.8
The process has worked successfully since GNS3 1.5

All projects seems to be working fine but when GNS3 VM starts the following modal dialog appears.
"The VMWware Tools power-on script did not run successfully 
in this virtual machine. If you have configured a custom
power-on script in this virtual machine, make sure
that it contains no errors.....'
When I close this dialog the GNS3 VM completes loading.  

When I close GNS3 application a similar modal dialog appears. Closing that dialog closed the VM and the GNS3 application.
Bottom line, based on cursory testing it appears not to be a show stopper but just a curiosity.
Hope my explanation makes sense.  Thanks!

**Comment by jtorri** [comment]
Not sure if significant but this behavior does not occur with GNS3 2.1.7

**Comment by e1ext** [comment]
I'm having the same issue after upgrading from 2.1.6 to 2.1.8, also I already tried the workaround from https://github.com/GNS3/gns3-gui/issues/2511 hoping that it would have different result on me.

Not only powering it on but powering off is also having issue

Is there any workaround to fix it?



Here's error when I close gns3 and vmware tried to shutdown the gns3 vm

"The request to power off this virtual machine failed because the corresponding VMWare tools script did not run successfully"

![image](https://user-images.githubusercontent.com/40454655/41705896-9891b12e-7564-11e8-935a-8c26f35e0e8d.png)

**Comment by JonathanFourie** [comment]
My two coppers worth here...

The difference between 2.1.6 and 2.1.8... 
2.1.6 contains the following files
in directory /etc/vmware-tools;
_poweroff-vm-default
poweron-vm-default
suspend-vm-default
tools.conf.dpkg-new
vm-support
poweroff-vm-default.dpkg-new
resume-vm-default
statechange.subr
tools.conf
tools.conf.old_
/etc/vmware-tools/scripts/vmware
_network
network.dpkg-new_

2.1.8 only contains the directories, no files
According to this thread, it may be a packaging bug in Ubuntu
https://github.com/vmware/open-vm-tools/issues/43

**Comment by jtorri** [comment]
Thanks JonathanFourie. That seemed to do the trick!

**Comment by e1ext** [comment]
Hi JonathanFourie,

You are right, the vmware-tools folder only contain folders, and I did as you suggested and it did the trick as jtorri said, no more power-on and off error

Thanks

**Comment by grossmj** [comment]
I reverted https://github.com/GNS3/gns3-vm/commit/0f9811cf8933b21f1cb2f9dd66d9536adf21a9ec which seems to have created this issue.

Hopefully everything will be fine for next release. Sorry for that.

**Comment by sajith-kumar** [comment]
I had the same issue with GNS3.VM.Workstation (2.1.9). But I followed the instruction given by @JonathanFourie , great it works.

**Comment by benvision1** [comment]
Thanks @JBSecvrity

**Comment by arys123** [comment]
thanks @JBSecvrity
it works
